cumulusci/salesforce_api/metadata.py

```diff
@@ -509,69 +483,6 @@ def _process_response(self, response):
         return self.status
 
 
-class ApiInstallVersion(ApiDeploy):
-
-    def __init__(self, task, version, purge_on_delete=False):
-        self.version = version
-        # Construct and set the package_zip file
-        if self.version.number:
-            self.package_zip = PackageZipBuilder(
-                self.version.package.namespace, self.version.number).install_package()
-        elif self.version.zip_url or self.version.repo_url:
-            if self.version.repo_url:
-                repo_url = self.version.repo_url
-                git_ref = self.version.branch
-                if installation_step.installation.git_ref:
-                    git_ref = installation_step.installation.git_ref
-                if installation_step.installation.fork:
-                    repo_url_parts = repo_url.split('/')
-                    repo_url_parts[3] = installation_step.installation.fork
-                    repo_url = '/'.join(repo_url_parts)
-                zip_url = '%s/archive/%s.zip' % (repo_url, git_ref)
-            else:
-                zip_url = self.version.zip_url
-            # Deploy a zipped bundled downloaded from a URL
-            try:
-                zip_resp = requests.get(zip_url)
-            except:
-                raise ValueError('Failed to fetch zip from %s' %
-                                 self.version.zip_url)
-            zipfp = StringIO.StringIO(zip_resp.content)
-            zipfile = ZipFile(zipfp, 'r')
-            if not self.version.subfolder and not self.version.repo_url:
-                zipfile.close()
-                zipfp.seek(0)
-                self.package_zip = base64.b64encode(zipfp.read())
-            else:
-                ignore_prefix = ''
-                if self.version.repo_url:
-                    # Get the top level folder from the zip
-                    ignore_prefix = '%s/' % zipfile.namelist()[0].split('/')[0]
-                # Extract a subdirectory from the zip
-                subdirectory = ignore_prefix + self.version.subfolder
-                subzip = zip_subfolder(
-                    zipfile, subdirectory, self.version.namespace_token, self.version.namespace)
-                subzipfp = subzip.fp
-                subzip.close()
-                subzipfp.seek(0)
-                self.package_zip = base64.b64encode(subzipfp.read())
-        super(ApiInstallVersion, self).__init__(
-            task, self.package_zip, purge_on_delete)
-
-
-class ApiUninstallVersion(ApiDeploy):
-
-    def __init__(self, task, version, purge_on_delete=True):
-        self.version = version
-        if not version.number:
-            self.package_zip = None
-        else:
-            self.package_zip = PackageZipBuilder(
-                self.version.package.namespace).uninstall_package()
-        super(ApiUninstallVersion, self).__init__(
-            task, self.package_zip, purge_on_delete)
-
```

Comment:
These are going away because they weren't used?

Reply:
Yes, I was trying to figure out how to get test coverage and realized there was no where that called this.  An earlier check, if I recall, already ensured this condition so it was impossible to have a test ever hit these lines.